### PR TITLE
Added Chapter 18

### PR DESCRIPTION
**Pull request by lehonma**
This is a rough conversion from DOCX.

### CODE REVIEW

**Comment by jonasmalacofilho** [comment]
- [x] table notes (depends on #25)
- [x] figure and table size adjustments

File: guide/18-fare_systems/181-fare-system-functionality.src [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+\col Low
+\col 1.1
+\col 0.9
+\row
+\col Off-board
+\col 1.1
+\col Level
+\col High
+\col 0.75\sup{1}
+\col 0.51\sup{1}
```
Comment (jonasmalacofilho) [inline guide/18-fare_systems/181-fare-system-functionality.src]: Check needed: was this really `0.51`/Colombia–Mexico?

**Comment by jonasmalacofilho** [comment]
We apparently hit a LFS issue with this maintainer commit to a fork.  : /

**Comment by jonasmalacofilho** [comment]
LFS issue bypassed...  Some idea why it failed, but no idea why it worked before!

File: guide/18-fare_systems/181-fare-system-functionality.src (hunk 1 above)
Comment (lehonma) [reply guide/18-fare_systems/181-fare-system-functionality.src]: No..it was 0.5^1. I'll fix that

**Comment by jonasmalacofilho** [comment]
We still need to fix figure/table sizes.

Besides that, the history of this needs to be rewritten (ideally when otherwise ready for the merge) to remove any EMF/GIF files from the non-LFS history.

A quick example of one way to do it on the CLI:

```
# checkout the PR branch
git checkout Chapter18
# create and checkout a copy
git checkout -b temp-chapter18-rebase-for-lfs-fix

# analyze the history to see which commits added EMF/GIF files
# search for EMF/GIF files being added (or remove) and take note of those commits
git log development..HEAD

# start a interactive rebase
# any commits that add (or modify) EMF/GIF files need to *edit*ed; all other can simply be *pick*ed
git rebase --interactive development

# depending on where you are on the rebase, you'll need to:
# a) ammend a commit so that it uses LFS for EMF/GIF
git commit --amend
# b) manually remove a (previously) deleted EMF/GIF file to fix a rebase conflict
# (since to git the LFS file and the non-LFS are different)
git rm guide/18-fare_systems/assets/image9.emf
git commit --amend
# c) let git continue with the rebase
git rebase --continue

# check the results
git log --oneline --decorate --graph development Chapter18 temp-chapter18-rebase-for-lfs-fix
# if all is well, reset the PR branch to the temporary branch
# BE CAREFUL to check what you're doing before the reset --hard
git checkout Chapter18
git reset --hard temp-chapter18-rebase-for-lfs-fix
```

An example of the resulting rebased branch exists in my fork: jonasmalacofilho/test-chapter18-rebased-for-lfs-fix.

File: guide/18-fare_systems/181-fare-system-functionality.src (hunk 1 above)
Comment (lehonma) [reply guide/18-fare_systems/181-fare-system-functionality.src]: Fixed on 19ff24d

**Comment by lehonma** [comment]
This chapter is ready for a rebase

**Comment by jonasmalacofilho** [comment]
ok, thanks

**Comment by lehonma** [comment]
My working-tree was rotten (heh), so I added my changes on pull request #81

**Pull request closed without merging**